src/main/java/com/maxi/calendar/model/User.java

```diff
@@ -0,0 +1,42 @@
+package com.maxi.calendar.model;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Table(name="user")
+@AllArgsConstructor @Getter @Setter
+public class User {
+	
+	@Id
+	@GeneratedValue(strategy = GenerationType.AUTO)
+	private int id;
+	
+	private String name;
+	private String role;
```

Comment:
Role should be an Enum. Possible values are: PM, QA, DEVELOPER, DEVOPS. Add values you think are relevant to this enum

Reply:
Roles enum was created